ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__edge.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  coreos-centos-stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.37.1
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile.image-service
```

Comment:
are we required to define this image both here and in the main config?

Reply:
Note: For the `assisted-image-service` this is more about keeping it consistent. The assisted-image-service wasn't slotted for the 4.11 OCP payload but I am trying to keep this as consistent as possible.